airbyte-webapp-e2e-tests/cypress/pages/connection/connectionPageObject.ts

```diff
@@ -1,12 +1,17 @@
 import { Connection } from "commands/api/types";
 import { getWorkspaceId } from "commands/api/workspace";
 
+const replicationTab = "div[data-id='replication-step']";
 const syncEnabledSwitch = "[data-testid='enabledControl-switch']";
 
-export const visitConnectionPage = (connection: Connection, tab = "") => {
+export const visit = (connection: Connection, tab = "") => {
   cy.intercept("**/web_backend/connections/get").as("getConnection");
   cy.visit(`/workspaces/${getWorkspaceId()}/connections/${connection.connectionId}/${tab}`);
   cy.wait("@getConnection", { timeout: 20000 });
 };
 
+export const goToReplicationTab = () => {
```

Comment:
I'm mildly tempted to say these all should be bundled into a class or object since an unsuspecting dev could just import `visit` from anywhere it's defined, and not use the `* as` import.
It's something we can catch in a code review but still worth considering. Maybe as a later PR?

Reply:
@krishnaglick As I was working with this I also think it's a better approach, and also it allows us to automatically import the object through auto-complete, where in this case, we have to write the import manually. Something for a later PR.